src/react-integration/__tests__/fixtures.ts

```diff
@@ -74,5 +74,5 @@ export const nested = [
 ];
 
 describe('fixtures', () => {
-  it('should pass', () => {})
+  it('should pass', () => {});
```

Comment:
?

Reply:
formatting fix. This was discussed in another pr as to why there's an empty test. (basically so I can put a file in __tests__